lib/index.ts

```diff
@@ -25,12 +20,7 @@ export function getCanonicalLocale(): string {
  * Returns the user's language
  */
 export function getLanguage(): string {
-    if (typeof OC === 'undefined') {
-        console.warn('No OC found')
-        return 'en';
-    }
-
-    return OC.getLanguage()
+    return document.documentElement.lang
```

Comment:
should this have a fallback to `'en'` too?

Reply:
There is none in the current implementation: https://github.com/nextcloud/server/blob/master/core/src/OC/l10n.js#L360=
But better safe than sorry, added.